### PR TITLE
added my name to contributor list

### PR DESCRIPTION
**Pull request by AqsaMemon**
I've added my name to the list. @aneagoie

### CODE REVIEW

**Review by sophiabrandt — CHANGES_REQUESTED** [review]
Can you please explain why the changes to the `README.md` file are required? This looks like a straight copy.

Also, please don't edit the archived file. Trying to merge conflicts in a file with 30,000 lines and more is a hassle.

Please fix the syntax errors in `CONTRIBUTORS.md`. We don't need the merge conflict markers there.

File: CONTRIBUTORS.md [hunk 1]
```diff
@@ -2944,12 +2944,6 @@
 
 - [@shubhamgoel90](https://github.com/shubhamgoel90)
 
-<<<<<<< HEAD
-
-=======
-
->>>>>>> 730949ce3e211ac66e330835d16a1001d87c7e09
-
```
Comment (AqsaMemon) [inline CONTRIBUTORS.md:2952]: done

Comment (sophiabrandt) [reply CONTRIBUTORS.md:2952]: Yes, but what about the other files? For example, `README.md` (see my requested changes/questions before)?
Also, please update your fork to the upstream repository, see https://github.com/zero-to-mastery/start-here-guidelines#a-guide-to-get-started Step 4.

**Comment by sophiabrandt** [comment]
Solved via PR https://github.com/zero-to-mastery/start-here-guidelines/pull/10009, thus closing here.

**Pull request closed without merging**